setup/markdownlint/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "laminas-continuous-integration-action-markdownlint",
+  "description": "Frozen dependencies required for installing a stable \\`markdownlint\\` binary",
+  "dependencies": {
+    "markdownlint-cli2": "0.3.2"
```

Comment:
Note: `markdownlint-cli2:0.4.0` is out, and it broke all our CI pipelines (in all repositories) because of our inattention to this lockfile problem here.